src/js/components/awardv2/contract/Recipient.jsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * Recipient.jsx
+ * Created by Kwadwo Opoku-Debrah 10/11/2018
+ **/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Building } from 'components/sharedComponents/icons/Icons';
+
+const propTypes = {
+    award: PropTypes.object
+};
+
+export default class Recipient extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            open: false
+        };
+
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+    handleClick() {
+        this.setState({ open: !this.state.open });
+    }
+
+    render() {
+        return (
+            <div className="award__col award-viz award-amounts">
+                <div className="award-viz__heading">
+                    <div className="award-viz__icon">
+                        <Building />
+                    </div>
+                    <h3 className="award-viz__title">
+                        Recipient
+                    </h3>
+                </div>
+                <hr />
+                <div className="award-amounts__content">
+                    <h4>{this.props.award.recipient.name}</h4>
```

Comment:
The `this.props.award` could be stored in a variable, this would make the code easier to read.